### PR TITLE
drf 315 added

### PR DESCRIPTION
**Pull request by MehrazRumman**

### CODE REVIEW

**Comment by MehrazRumman** [comment]
@auvipy  you can review now .

File: tox.ini [hunk 1]
```diff
@@ -2,7 +2,7 @@
 envlist = py{38,39}-django{22}-drf{311,312}
           py{38,39,310,311,312}-django{32}-drf{312,313,314}
           py{38,39,310,311,312}-django{42}-drf{314}
-          py{310,311,312}-django{51}-drf{314}
+          py{310,311,312}-django{51}-drf{314,315}
```
Comment (auvipy) [inline tox.ini:5]: This version should also be tested with dj 4.2 lts

**Review by auvipy — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (MehrazRumman) [reply tox.ini:5]: ok I am adding it for django 4.2 also

**Comment by MehrazRumman** [comment]
@auvipy  you can review the latest changes !

**Review by auvipy — APPROVED** [review]

**Pull request merged**